app/src/main/kotlin/com/mapbox/vision/teaser/utils/FileUtils.kt

```diff
@@ -0,0 +1,25 @@
+package com.mapbox.vision.teaser.utils
+
+import java.io.File
+import java.lang.RuntimeException
+
+object FileUtils {
+
+    fun deleteFolder(folder: File) {
```

Comment:
Kotlin has the default extension `public fun File.deleteRecursively()`, can it fit instead of manual impl?

Reply:
yeah, I'll try

Reply:
fixed